dcat/rdf/dcat3.ttl

```diff
@@ -1730,31 +1795,32 @@ dcat:version
   a rdf:Property ;
   a owl:DatatypeProperty ; 
   owl:equivalentProperty pav:version ;
-  rdfs:label "version"@en ;
-  rdfs:label "versión"@es ;
-  rdfs:label "versione"@it ;
   rdfs:comment "The version indicator (name or identifier) of a resource."@en ;
   rdfs:comment "El indicador de versión (nombre o identificador) del recurso."@es ;
   rdfs:comment "L'indicatore di versione (un nome o un identificatore) di una risorsa."@it ;
-  skos:definition "The version indicator (name or identifier) of a resource."@en ;
-  skos:definition "El indicador de versión (nombre o identificador) del recurso."@es ;
-  skos:definition "L'indicatore di versione (un nome o un identificatore) di una risorsa"@it ;
-  skos:scopeNote "DCAT does not prescribe how a version name / identifier should be specified, and refers for guidance to [DWBP]'s Best Practice 7: Provide a version indicator."@en ;
-  skos:scopeNote "DCAT "@es ;
-  skos:scopeNote "DCAT non prescrive come un nome o identificatore di versione dovrebbe essere specification, and fa riferimento alle linee guida indicate in [DWBP] Best Practice 7: Provide a version indicator."@it ;
+  rdfs:isDefinedBy <https://www.w3.org/TR/vocab-dcat-3/> ;
+  rdfs:label "version"@en ;
+  rdfs:label "versión"@es ;
+  rdfs:label "versione"@it ;
   skos:changeNote "New property added in DCAT 3."@en ;
   skos:changeNote "Nueva propiedad agregada in DCAT 3."@es ;
   skos:changeNote "Nová vlastnost přidaná ve verzi DCAT 3."@cs ;
   skos:changeNote "Nuova proprietà aggiunta in DCAT 3."@it ;
   skos:changeNote "Ny egenskab tilføjet i DCAT 3."@da ;
+  skos:definition "The version indicator (name or identifier) of a resource."@en ;
+  skos:definition "El indicador de versión (nombre o identificador) del recurso."@es ;
+  skos:definition "L'indicatore di versione (un nome o un identificatore) di una risorsa"@it ;
+  skos:scopeNote "DCAT does not prescribe how a version name / identifier should be specified, and refers for guidance to [DWBP]'s Best Practice 7: Provide a version indicator."@en ;
+  skos:scopeNote "DCAT  "@es ;
```

Comment:
@agbeltran , the Spanish translation is missing here.

Reply:
thanks @andrea-perego - I added the missing translation for you to consider

Reply:
Thanks, @agbeltran .